spec/features/idv/in_person_spec.rb

```diff
@@ -319,9 +321,10 @@
 
       perform_in_browser(:desktop) do
         user = sign_in_and_2fa_user
-        complete_doc_auth_steps_before_send_link_step
-        fill_in :doc_auth_phone, with: '415-555-0199'
-        click_idv_continue
+        complete_doc_auth_steps_before_upload_step
+        fill_in :doc_auth_phone, with: ''
+        fill_in :doc_auth_phone, with: phone_number
```

Comment:
Same comment as for hybrid_flow_spec.rb  about filling this field twice.

Also if you want to set a variable for phone_number, please place it much closer to where it is used, not over 300 lines away.